charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -15,12 +15,20 @@ maintainers:
     email: miroslav.hadzhiev@gmail.com
 name: kube-prometheus-stack
 sources:
+  - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-  - https://github.com/prometheus-operator/prometheus-operator
-version: 9.4.1
+version: 9.4.2
 appVersion: 0.38.1
 tillerVersion: ">=2.12.0"
 home: https://github.com/prometheus-operator/kube-prometheus
 keywords:
 - operator
 - prometheus
+- kube-prometheus
+annotations:
+  "artifacthub.io/operator": true
```

Comment:
@scottrigby seems you need to make this as `string`. guess this is evaluated against https://github.com/helm/chart-testing/blob/master/etc/chart_schema.yaml#L17 and failing.

Reply:
We could change the yamale definition for `ct`, but You're right: https://github.com/helm/helm/blob/master/pkg/chart/metadata.go#L58
```go
	Annotations map[string]string `json:"annotations,omitempty"`
```

I'll update the Artifact Hub docs.